video_xblock/static/js/studio-edit.js

```diff
@@ -12,6 +12,11 @@ function StudioEditableXBlock(runtime, element) {
     // Studio includes a copy of tinyMCE and its jQuery plugin
     var tinyMceAvailable = (typeof $.fn.tinymce !== 'undefined');  // TODO: Remove TinyMCE
     var datepickerAvailable = (typeof $.fn.datepicker !== 'undefined'); // Studio includes datepicker jQuery plugin
+    var $enabledLabel = $('div.custom-field-section-label:contains("Enabled transcripts")');
```

Comment:
Very bad selector. Titles will support translations in a feature. Please use some className for the selector.